index.html

```diff
@@ -190,7 +190,7 @@ <h3 class="card-title align-middle"> <i class="bi bi-terminal me-2"></i> Develop
                     <li class="pb-3"> Code mostly without hardware dependence </li>
                     <li class="pb-3"> Code once, run on 8-bit  (e.g., Arduino Mega 2560), 16-bit  (e.g.,
                       MSP430), and 32-bit platforms </li>
-                    <li class="pb-3"> Benefit from POSIX-compliant components </li>
+                    <li class="pb-3"> Benefit from components with a high level of POSIX compliance </li>
```

Comment:
I don't really get what "components" in the context of POSIX compliance means. If that refers to the modularity of RIOT, I think this a bit out of place here: POSIX only specifies the C API (one some command line utilities). Whether the implementation of that API is modular or not, is from the POSIX point of view an opaque implementation detail.

That said, I agree that the modularity is a key feature of RIOT. But IMO it is better to point that out separately of POSIX. Especially since you can take advantage of the modularity independent of using the POSIX APIs.

Reply:
To be honest, the previous wording was already vague -- it doesn't say that everything is fully compliant. The new wording (e.g., "high-level") is confusing.

Just to be clear: @maribu original comment related to the text `Benefit from POSIX compliance` which is a different statement compared to `Benefit from POSIX-compliant components`.

I don't see that `Benefit from POSIX-compliant components` is lying.

Reply:
But still, what are POSIX compliant components? An OS is POSIX compliant, not a component.

Reply:
Then, let's go for `Benefit from available POSIX calls` ?

Reply:
RIOT doesn't contain POSIX calls. The application contains the calls. RIOT provides the API to be called.

Reply:
i don't see a conflict. RIOT is more than the kernel, for example.

we don't need to make it more complicated than it is. 

if there are no better proposals on the table, we should either go for `Benefit from POSIX-compliant components` or `Benefit from available POSIX calls`.

Reply:
Well, components cannot be POSIX compliant. It is a standard for OS. This rules `Benefit from POSIX-compliant components` out.

> i don't see a conflict. RIOT is more than the kernel, for example.

The POSIX standard is about the C-API (and some shell applications) an OS provides. It doesn't say anything about the kernel at all. In fact, a kernel by itself cannot be POSIX compliant, as it is missing the C-API (and the applications).

@waehlisch: I think this discussion would be more productive if you would understand what POSIX compliance actually is. Maybe check Wikipedia?

Reply:
yes, sure, i will check Wikipedia.